internal/cephfs/controllerserver.go

```diff
@@ -830,54 +830,19 @@ func (cs *ControllerServer) CreateSnapshot(
 	}
 	defer cs.VolumeLocks.Release(sourceVolID)
 	snapName := req.GetName()
-	sid, snapInfo, err := store.CheckSnapExists(ctx, parentVolOptions, cephfsSnap, cs.ClusterName, cs.SetMetadata, cr)
+	sid, _, err := store.CheckSnapExists(ctx, parentVolOptions, cephfsSnap, cs.ClusterName, cs.SetMetadata, cr)
```

Comment:
This is the only usage of `CheckSnapExists()`. Please add a 2nd commit to cleanup the function, and not return the unused `snapInfo` anymore.